tendrl/gluster_integration/sds_sync/__init__.py

```diff
@@ -110,7 +102,8 @@ def _run(self):
                                     hostname=peers['peer%s.primary_hostname' % index],
                                     state=peers['peer%s.state' % index]
                                 )
-                            peer.save()
+                            peer.save(ttl=int(
+                                NS.config.data.get("sync_interval", 10)) + SYNC_TTL)
```

Comment:
Calculate `int(NS.config.data.get("sync_interval", 10)) + SYNC_TTL)` only once in this loop